src/topology/bornology/basic.lean

```diff
@@ -0,0 +1,134 @@
+/-
+Copyright (c) 2022 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+import order.filter.cofinite
+
+/-!
+# Basic theory of bornology
+
+We develop the basic theory of bornologies. Instead of axiomatizing bounded sets and defining
+bornologies in terms of those, we recognize that the cobounded sets form a filter and define a
+bornology as a filter of cobounded sets which contains the cofinite filter.  This allows us to make
+use of the extensive library for filters, but we also provide the relevant connecting results for
+bounded sets.
+-/
```

Comment:
It would be good if the documentation had a "main definitions" section with `bornology`, `bornology.is_bounded`, and `bornology.is_cobounded`, along with a mention that these latter predicates should be preferred over the `bornology.cobounded` filter.

Reply:
done